uxarray/__init__.py

```diff
@@ -1,2 +1,3 @@
 from .grid import *
 from .dataset import *
+from .helpers import *
```

Comment:
Do you really wnat all the helpers to be exposed publicly?

Reply:
It seemed possible and I thought it might be useful for some projects (giving our projects added utility), can be removed also (fine either way).

Reply:
The way we have it set up now, some of the functions in helper are available through the user api: https://uxarray.readthedocs.io/en/integration/user_api/index.html

Do you think we should change this?

Reply:
@anissa111 Hm... do those work with just `import uxarray` . Seems like that might require a separate `import uxarray.helpers`. 

You could do `from . import helpers` here and `uxarray.helpers` would always be available